resources.go

```diff
@@ -1046,11 +990,10 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_application_security_group": {Tok: azureDataSource(azureNetwork, "getApplicationSecurityGroup")},
 			"azurerm_recovery_services_vault":    {Tok: azureDataSource(azureRecoveryServices, "getVault")},
 			"azurerm_redis_cache":                {Tok: azureDataSource(azureRedis, "getCache")},
-
-			"azurerm_resource_group": {Tok: azureDataSource(azureCore, "getResourceGroup")},
-			"azurerm_snapshot":       {Tok: azureDataSource(azureCompute, "getSnapshot")},
-			"azurerm_subnet":         {Tok: azureDataSource(azureNetwork, "getSubnet")},
-			"azurerm_route_table":    {Tok: azureDataSource(azureNetwork, "getRouteTable")},
+			"azurerm_resource_group":             {Tok: azureDataSource(azureCore, "getResourceGroup")},
+			"azurerm_snapshot":                   {Tok: azureDataSource(azureCompute, "getSnapshot")},
```

Comment:
aside, do we think azureCompute is the right namespace for this resource?  i honestly don't know.  but 'snapshot' sounds odd there.

Reply:
Resolved.  Checked tf and this is where it lives.